src/components/interfaces/HMI_API.xml

```diff
@@ -4731,12 +4731,42 @@
     </param>
 </function>
 
-
 <function name="OnInteriorVehicleData" functionID="OnInteriorVehicleDataID" messagetype="notification">
   <param name="moduleData" type="Common.ModuleData">
   </param>
 </function>
 
+<function name="ButtonPress" functionID="ButtonPressID" messagetype="request">
```

Comment:
@AKalinich-Luxoft `ButtonPress` already present in HMI_API and MOBILE_API updated according to the latest version
9402dc90b74d4c3d4932b95768ab1514f813f50d